hutch_python/tests/test_happi.py

```diff
@@ -63,3 +63,40 @@ def test_get_lightpath():
     # Check that we created a valid BeamPath with no inactive objects
     assert obj.name == 'TST'
     assert len(obj.devices) == 3
+
+
+# run test_happi_objs() without the excluded devices
+@conftest.requires_lightpath
+def test_happi_objs_without_exclude_devices():
```

Comment:
One functional thing including this test does is act as a foil to the next test, so if the base case changes then this test fails too so we know that the exclude case didn't fail on its own.

Maybe this would be clearer (and more useful to future us) if they were part of the same test sharing resources and comparing results.